cli/tools/repl/editor.rs

```diff
@@ -249,69 +249,79 @@ fn validate(input: &str) -> ValidationResult {
   let mut in_template = false;
   let mut div_token_count_on_current_line = 0;
   let mut last_line_index = 0;
-
-  for item in deno_ast::lex(input, deno_ast::MediaType::TypeScript) {
-    let current_line_index = line_info.line_index(item.range.start);
+  let mut queued_validation_error = None;
```

Comment:
The change here fixes another regression due to swc not having regex literals as tokens anymore. A pty test pointed this out.